Makefile

```diff
@@ -594,21 +622,30 @@ test: test-all	## alias for 'test-all' target
 .PHONY: test-all
 test-all: install-dev install test-only  ## run all tests combinations
 
+.PHONY: test-all
+test-all: install-dev install test-only  ## run all tests combinations, except Geoserver requests
+
 .PHONY: test-only
 test-only:  ## run all tests, but without prior dependency check and installation
 	@echo "Running tests..."
-	@bash -c '$(CONDA_CMD) pytest tests -vv --junitxml "$(APP_ROOT)/tests/results.xml"'
+	@bash -c '$(CONDA_CMD) pytest tests -vv -m "not geoserver" --junitxml "$(APP_ROOT)/tests/results.xml"'
```

Comment:
I think this `target` should remain "run all" regardless of applied tags.

Reply:
So leave this target to "run all" and create a new target that would then be the one called by Github actions?

Reply:
There could be separate job/matrix entries for api, cli, geoserver, etc.

Reply:
Would using `SPEC="not geoserver" test-custom` work in the context of Github actions? Could also be an alternative, especially for the python 3.9 test case if we don't want too much of an increase of checks in the report (would add 6 to cover the other markers).

Having separate entries for each marker in the main matrix could be helpful though, providing a quick view of what class of tests failed right off the bat.

Reply:
If you call specifically that spec often, a specific target is better. 
It is really based on usage convenience. 

One-of entry in the CI with `SPEC=...` is also valid, but you might need to update the test command since it expect only a target name: https://github.com/Ouranosinc/cowbird/blob/master/.github/workflows/tests.yml#L100-L101.

The only reason it is worth splitting by test markers in the test matrix is if some tests take specifically longer job-run time compared to others. If the full `test` target takes less or similar time to `checks`, I would just leave them all together. There is a limit on parallel jobs anyway (5 I think?), so over splitting won't make it faster. 

Other side note: 

This item: https://github.com/Ouranosinc/cowbird/blob/master/.github/workflows/tests.yml#L57
could be replaced by `[test-only]` since `install-dev` is already processed some steps before 
https://github.com/Ouranosinc/cowbird/blob/master/.github/workflows/tests.yml#L100-L101
(save up time of re-valiation of packages).